packages/quantic/force-app/main/default/lwc/quanticNumericFacet/__tests__/quanticNumericFacet.test.js

```diff
@@ -229,11 +409,444 @@ describe('c-quantic-numeric-facet', () => {
         expect(facetContent).toBeNull();
       });
     });
+  });
 
-    afterAll(() => {
-      functionsMocks.buildNumericFacet.mockReturnValue(
-        numericFacetControllerMock
+  describe('when the component is loading', () => {
+    beforeEach(() => {
+      searchStatusState = {
+        ...initialSearchStatusState,
+        isLoading: true,
+        hasError: false,
+        firstSearchExecuted: false,
+      };
+    });
+
+    it('should display the placeholder component', async () => {
+      const element = createTestComponent();
+      await flushPromises();
+
+      const placeholder = element.shadowRoot.querySelector(
+        selectors.placeholder
       );
+      expect(placeholder).not.toBeNull();
+    });
+
+    describe('when the interface has an error', () => {
+      beforeEach(() => {
+        searchStatusState = {
+          ...initialSearchStatusState,
+          isLoading: true,
+          hasError: true,
+          firstSearchExecuted: false,
+        };
+      });
+
+      it('should not display the placeholder component', async () => {
+        const element = createTestComponent();
+        await flushPromises();
+
+        const placeholder = element.shadowRoot.querySelector(
+          selectors.placeholder
+        );
+        expect(placeholder).toBeNull();
+      });
+    });
+  });
+
+  describe('when the component is ready', () => {
+    describe('when the facet has values', () => {
+      beforeEach(() => {
+        facetState = {
+          ...initialFacetState,
+          values: exampleFacetValues,
+        };
+      });
+
+      it('should display the facet card when the facet values have results', async () => {
+        const element = createTestComponent();
+        await flushPromises();
+
+        const cardContainer = element.shadowRoot.querySelector(
+          selectors.cardContainer
+        );
+        expect(cardContainer).not.toBeNull();
+      });
+
+      it('should not display the facet card when the facet values have no results', async () => {
```

Comment:
```suggestion
      it('should display the facet card when those facet values have results', async () => {
        const element = createTestComponent();
        await flushPromises();

        const cardContainer = element.shadowRoot.querySelector(
          selectors.cardContainer
        );
        expect(cardContainer).not.toBeNull();
      });

      it('should not display the facet card when those facet values have no results', async () => {
```

Minor suggestion, but I think this makes reading this test a bit less confusing IMO